src/cxy_ctl.erl

```diff
@@ -601,9 +601,22 @@ execute_wrapper(Mod, Fun, Args, Task_Type, Max_History, Start, Spawn_Or_Inline,
     end.
 
 -spec fail_wrapper(spawn | inline, any(), any()) -> no_return().
-fail_wrapper(spawn,  Call_Data, Stacktrace) -> erlang:error(spawn_failure,  [Call_Data, Stacktrace]);
-fail_wrapper(inline, Call_Data, Stacktrace) -> exit       ({inline_failure, [Call_Data, Stacktrace]}).
-
+fail_wrapper(spawn,  Call_Data, Stacktrace) -> erlang:error(spawn_failure,  [sanitize(Call_Data), Stacktrace]);
+fail_wrapper(inline, Call_Data, Stacktrace) -> exit       ({inline_failure, [sanitize(Call_Data), Stacktrace]}).
+
+% timeout on nested gen_server:call/2,3 - most callers are
+% gcache modules trying to replenish cache (create_key_value)
+% in the deepest callstack, redis connection goes away for a bit
+
+% search for 'spawn_failure' or 'inline_failure' in the logs, you will see more
+% pattern match for specific case - trade off between
+% valuable debugging information vs. private customer's data
+sanitize({exit, {timeout, {gen_server, call, _}},
+    {creating_new_value_with, gcache_auth, {check_auth, _}}}) ->
```

Comment:
so this fix is specific for gcache_auth in cxy_ctrl, instead of generic solution?

Reply:
yeah, I could have matched for gen server call timeout, function clause, badmatch, and that would hide almost everything. I am not sure that is a good idea though? it will leave us with bare stacktrace like cxy_ctl execute this and that. and that is not enough of information for debugging?